source/_less/patternfly-site.less

```diff
@@ -1442,3 +1442,41 @@ th {
     }
   }
 }
+
+.pforg-code__title-row {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 25px;
+}
+
+.pforg-code__core-example-title {
+  margin: 0;
+  padding: 0;
+}
+
+.pforg-code__showcase-link-list {
+  list-style: none;
+  margin: 0;
+  padding: 0 0 25px 0;
+  display: flex;
+  flex-wrap: wrap;
+
+  border-bottom: 1px solid #D5D5D5;
+}
+
+.pforg-code__showcase-link-item {
+  padding-right: 4rem;
+  display: block;
```

Comment:
What is the display:block; for?

Reply:
hmmm I actually forgot to remove that. I don't need it since I have display: flex; on the parent.  Will go ahead and remove.